Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -2824,6 +2825,13 @@ private void OnIbAutomaterOutputDataReceived(object sender, OutputDataReceivedEv
 
             Log.Trace($"InteractiveBrokersBrokerage.OnIbAutomaterOutputDataReceived(): {e.Data}");
 
+            // login failed
+            if (e.Data.Contains("Login failed"))
+            {
+                _loginFailed = true;
+                _ibAutomaterInitializeEvent.Set();
+            }
+
             // an existing session was detected and IBAutomater clicked the "Exit Application" button
             if (e.Data.Contains("Existing session detected"))
```

Comment:
I think these could be `else if` statements?
Wondering if these data messages are specific per language setting? Which is okay, just maybe worth noting it for Lean users somehow/where

Reply:
- `else if` -- sure, they **should** be, will fix 👍 

- languages -- actually the IB Gateway login dialog contains a Language button/submenu (which a LEAN user could use to change from English to one of other 8 languages). 
For now, we support only the English language which is not an issue in the cloud, since IBAutomater will never touch that button 😃